packages/api/tests/endpoints/test-granules.js

```diff
@@ -446,182 +463,212 @@ test.serial('default paginates correctly with search_after', async (t) => {
   t.not(meta.searchContext === newMeta.searchContext);
 });
 
-test.serial('CUMULUS-911 GET without pathParameters and without an Authorization header returns an Authorization Missing response', async (t) => {
```

Comment:
Not-nit -- this change impedes our ability to fold at this level and casually browse test titles.    We should revert these.

Reply:
https://github.com/nasa/cumulus/pull/3225/commits/ea12a2846db17b78e9107a5109a445018d7ddfc8